src/app-init/apiManager.js

```diff
@@ -1,12 +1,13 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { ApiProvider } from '@entando/apimanager';
+import { fetchPermissions } from 'state/permissions/actions';
+import { fetchUserAuth, clearCurrentUserAuth } from 'state/current-user-auth/actions';
```

Comment:
Why `UserAuth` if it's about group permission? This name is misleading

Reply:
meant for "UserAuthorities" but I made some changes by moving all them on `permission` state